modules/document_repository/php/document_repository.class.inc

```diff
@@ -310,6 +309,7 @@ class Document_Repository extends \NDB_Menu_Filter
         $query    = "SELECT * FROM document_repository_categories".
                     " ORDER BY parent_id ASC";
         $dbresult = $DB->pselect($query, array());
+        $tree     = array();
```

Comment:
See https://github.com/aces/Loris/pull/3656. It looks like this should be an object, not an array.

Reply:
Yes, see line 326. Its properties are accessed as an object.

Reply:
Updated the $tree to be an object as in #3656